### PR TITLE
plugins.douyutv: fix room id regex

### PR DESCRIPTION
**Pull request by steven7851**
from https://github.com/soimort/you-get/issues/1657

### CODE REVIEW

File: src/streamlink/plugins/douyutv.py [hunk 1]
```diff
@@ -109,9 +108,11 @@ def _get_streams(self):
         res = http.get(MAPI_URL.format(channel))
         room = http.json(res, schema=_room_schema)
         if not room:
+            self.logger.info("API error")
```
Comment (gravyboat) [inline src/streamlink/plugins/douyutv.py]: Can we be more descriptive with this error? `API error` isn't very helpful for users.

File: src/streamlink/plugins/douyutv.py [hunk 2]
```diff
@@ -109,9 +108,11 @@ def _get_streams(self):
...
             return
 
         if room["show_status"] != SHOW_STATUS_ONLINE:
+            self.logger.info("Channel offline now!")
```
Comment (gravyboat) [inline src/streamlink/plugins/douyutv.py]: Can we make this more neutral? `Channel is offline` or something would be fine.

File: src/streamlink/plugins/douyutv.py [hunk 3]
```diff
@@ -109,9 +106,11 @@ def _get_streams(self):
         res = http.get(MAPI_URL.format(channel))
         room = http.json(res, schema=_room_schema)
         if not room:
+            self.logger.info("Not a valid room url.")
             return
 
-        if room["show_status"] != SHOW_STATUS_ONLINE:
+        if room["show_status"] != 1:
+            self.logger.info("Stream current unavailable.")
```
Comment (beardypig) [inline src/streamlink/plugins/douyutv.py]: You should change this to `Stream currently unavailable`

**Comment by beardypig** [comment]
Looks good, just one suggested change for grammar :)

**Comment by beardypig** [comment]
Looks good to me now :)

**Comment by beardypig** [comment]
What d'ya think @gravyboat ?

**Comment by gravyboat** [comment]
Yeah I'm good with this, thanks @steven7851.

**Pull request merged**